src/App.vue

```diff
@@ -126,6 +130,8 @@ async function setupAxiosInterceptors(axios: any) {
     if (isLoggedIn.value) {
       if (!request.headers) request.headers = {};
       request.headers.Authorization = "Bearer " + (await fetchAuthSession()).tokens?.idToken;
+    } else if (import.meta.env.VITE_HOSTING_MODE === "development") {
```

Comment:
This should be the default behavior for security reasons. Invert logic to check for: 
if not development mode -> show page
otherwise -> return to login page

Reply:
Logic looks ok to me. If logged in everything should work on any mode. Only if not logged in and in dev hosting mode are you forced to login